bin/configs/kotlin-server-jaxrs-spec-array-response.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: kotlin-server
+outputDir: samples/server/others/kotlin-server/jaxrs-spec-array-response
```

Comment:
can you please update github workflow with the new folder so that the CI will test these moving forward?

```
.github/workflows/samples-kotlin-server-jdk17.yaml
.github/workflows/samples-kotlin-server.yaml
```

Reply:
Done!

Reply:
After running the build it fails on jdk 17. To make it work I need to do some more work on the template files which is unrelated to this change. @wing328 could I make those changes in a new PR so we don't scope creep on this one? 
The sample runs fine on Java 11

Reply:
sure. i'll get it merged